modules/minileven/theme/pub/minileven/functions.php

```diff
@@ -237,5 +245,15 @@ function minileven_get_gallery_images() {
 function minileven_show_featured_images() {
 	$enabled = ( is_home() || is_search() || is_archive() ) ? true : false;
 
+	/**
+	 * Filter where featured images are displayed in the Mobile Theme.
+	 * 
+	 * For example, by defining $enabled as above, featured images will display 
```

Comment:
I'm trying to think how to reword this, thinking of it as it would displayed on a parsed docs site that doesn't have the comment inline with the code before or after it. I'm blanking though.